libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/PluggableAreaChart.tsx

```diff
@@ -60,6 +59,7 @@ import {
     modifyBucketsAttributesForDrillDown,
     reverseAndTrimIntersection,
 } from "../drillDownUtil";
+import { IDrillDownDefinition } from "../../../dashboardEmbedding/types";
```

Comment:
This is a bit sketchy, plugVis should probably not "reach" into dashboardEmbedding. Should we move that type in the root of the pacakge or something?